web/packages/api/src/status.ts

```diff
@@ -67,8 +67,8 @@ export type Sovereign = { name: string; account: string; balance: bigint; type:
 export const BlockLatencyThreshold = {
     // Syncing beefy finality update every 4 hours(2400 blocks) so we set 3000 blocks at most.
     ToEthereum: 3000,
-    // Syncing beacon finality update every 4 hours(1200 blocks) so we set 1500 blocks at most.
-    ToPolkadot: 1500,
+    // Syncing beacon finality update every 6.4 minutes(32 blocks) so we set 128 blocks (4 epochs) at most.
+    ToPolkadot: 128,
```

Comment:
Shouldn't we add a small buffer to give the relayer some time to sync?

Reply:
Technically, we should never be more than 3 epochs behind. So I have added 1 epoch buffer in the alarm to allow for syncing.  I have been keeping an eye on the status dashboard and hardly see the latency go above 20 minutes. The alarm latency here is set to 25.6 minutes. 

What do you think is an acceptable latency?